frontend/app/routes/public/apply/$id/adult-child/contact-information.tsx

```diff
@@ -177,6 +177,8 @@ export async function action({ context: { appContainer, session }, params, reque
       ...val,
       homePostalCode: val.homeCountry && val.homePostalCode ? formatPostalCode(val.homeCountry, val.homePostalCode) : val.homePostalCode,
       mailingPostalCode: val.mailingCountry && val.mailingPostalCode ? formatPostalCode(val.mailingCountry, val.mailingPostalCode) : val.mailingPostalCode,
+      phoneNumber: val.phoneNumber,
+      phoneNumberAlt: val.phoneNumberAlt,
```

Comment:
I wouldn't think this would be an issue since `val` is spreaded before. If `phoneNumber` and `phoneNumberAlt` are present then they should copied over. I will have to debug it locally.

Reply:
I've tested and it works where `val` is spreaded.